Sources/MongoSwift/SDAM.swift

```diff
@@ -57,40 +65,35 @@ public struct ServerDescription {
     /// The hostname or IP and the port number that the client connects to. Note that this is not the
     /// server's ismaster.me field, in the case that the server reports an address different from the
     /// address the client uses.
-    public let connectionId: ConnectionId
-
-    /// The last error related to this server.
-    public let error: MongoError? = nil // currently we will never set this
+    public let address: Address
 
-    /// The duration of the server's last ismaster call.
+    /// The duration in milliseconds of the server's last ismaster call.
     public var roundTripTime: Int64?
```

Comment:
are all of these just `var`s so we can set them outside of the `init` in `parseIsMaster`? or is it possible I did this so the values could change over time in a true SDAM implementation? 
(I realize I wrote all of this code but I have no recollection of what I was thinking...)

Reply:
I remember thinking about this before break but I don't remember what conclusion I came to. `var` -> `let` is definitely a breaking change, but the other way I think should be fine. Do you think it's worth making these constants here? In the currently implementation they definitely shouldn't be mutated by the user or us.

Reply:
Updated them all to be `let`.